src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();

```

Comment:
Consider using a single instance of `Random` instead of creating a new one each time the `getRandomBall()` method is called. You can declare it as a private static final field in the `Lottery` class.
Same will be applicable for ColorSupplier once you will rework static method to non-static